src/main/java/step4/vehicle/Car.java

```diff
@@ -0,0 +1,39 @@
+package step4.vehicle;
+
+import step4.util.NumberGenerator;
+
+public class Car {
+    private static final int MOVE_CONDITION = 4;
+
+    private int location = 1;
+    private String name;
```

Comment:
```suggestion
    private final String name;
```
이름은 불변한 값이므로 final을 달아주면 좋을 것 같네요